src/main/java/io/confluent/connect/jdbc/sink/BufferedRecords.java

```diff
@@ -153,6 +152,12 @@ public BufferedRecords(
         );
       }
     }
+    
+    // set deletesInBatch if schema value is not null
```

Comment:
"schema value" or "record value"?

Reply:
```suggestion
    // set deletesInBatch if the record value is null, but schema value was non-null
```

Reply:
Actually, re-reading this it makes more sense, it just leaves out the record-value because that's already in the code itself.

Reply:
Maybe this does make sense, given that the null-schema case is handled above this line, and that when we get here we know the value schema is not null.